recipes/hydra-multi/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -eu -o pipefail
+
+mkdir -p $PREFIX/bin
+make all CXX="${CXX}" \
+    ZLIB_PATH="${PREFIX/lib}" \
+    CXXFLAGS="${CXXFLAGS}" \
+    LIBS="${LDFLAGS} -lm -lz -lpthread"
+#    CC="${CC}" \
+#    CXX="${CXX}" \
+#    CPPFLAGS="${CPPFLAGS}" \
+#    CFLAGS="${CFLAGS}" \
+#    LDFLAGS="${LDFLAGS} -lz" \
+
+chmod 755 hydra-multi.sh
+chmod 755 scripts/hydraToBreakpoint.py
+sed 's[/usr/local/bin/python2.6[/usr/bin/env python[' -i scripts/frequency.py
```

Comment:
Did you create an issue or pull request ton integrate this sed in upstream ?

Reply:
This package is not maintained for a very long time, I opened an issue for creating a release to no avail.